burstphoto/ContentView.swift

```diff
@@ -326,12 +324,22 @@ struct MyDropDelegate: DropDelegate {
                 DispatchQueue.main.async { app_state = .main }
             } catch AlignmentError.inconsistent_extensions {
                 my_alert.title = "Inconsistent formats"
-                my_alert.message = "The dropped files heve inconsistent formats. Please make sure that all images are DNG files."
+                my_alert.message = "Please make sure that all images have the same format."
                 my_alert.show = true
```

Comment:
Hmm. Technically of course you could have a single burst composed of different formats. But I'm afraid that when this happens, it's more likely to be an error rather than being intentional.

For example, you capture 5 ARWs, convert all of them to DNG (because you think it's still necessary) and then you drag all 10 images into the app. Or you drag your 5 ARWs together with the output DNG. From the app's perspective, these are all valid bursts…

I'm not aware of any good use case for this feature – if you can think of one, please let me know.

Reply:
I agree that it's very likely an error since I can't really think of a good case now.

I see that it gets called before converting trying to convert all files to dng, that's my bad I got turned around and thought it was after.